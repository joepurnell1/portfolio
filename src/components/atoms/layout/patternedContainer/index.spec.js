import React from 'react';
import renderer from 'react-test-renderer';
import ColouredContainer from './index';

describe('patterned container component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <ColouredContainer>When Im watchin my TV and a man comes on and tells me</ColouredContainer>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
