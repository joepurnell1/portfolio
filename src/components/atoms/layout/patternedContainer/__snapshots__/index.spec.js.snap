// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`patterned container component should render default as expected 1`] = `
<div
  className="sc-bwzfXH uTxCW sc-bdVaJa dWTsUi"
>
  When Im watchin my TV and a man comes on and tells me
</div>
`;
